plugins/lime-plugin-admin/src/adminApi.js

```diff
@@ -1,19 +1,2 @@
-import { map } from 'rxjs/operators';
-
-export const login = (sid, api, auth) =>
-	api.call(sid, 'session', 'login', Object.assign({}, auth, { timeout: 5000 })).pipe(
-		map(result => {
-			try {
-				if (typeof result.ubus_rpc_session === 'undefined') {
-					throw new Error('Login error');
-				}
-			}
-			catch (error){
-				// do nothing
-			}
-			return result.ubus_rpc_session;
-		})
-	);
-
-export const changeConfig = (api, sid, config) =>
-	api.call(sid, 'lime-utils', 'change_config', { name: config.hostname, ip: config.ip });
+export const changeConfig = (api, config) =>
```

Comment:
if we are assuming that the context manages the authentification, these call should handle the unauthorization through an exception and a try/catch, with a global catch-all.
Responsability-wise I think it would be the best.